kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -76,13 +76,21 @@ export default [
       store.commit('manageSync/RESET_STATE');
     },
   },
+
   {
     name: PageNames.FACILITY_CONFIG_PAGE,
     path: '/settings',
     handler: () => {
       showFacilityConfigPage(store);
     },
   },
+  {
```

Comment:
remove this route

Reply:
Done, I had left it from the previous specs